File: ExercicioCozinha/Funcionario.java
package SquadraNewThinkers_1T2021.ExercicioCozinha;

public class Funcionario {
    private String nome;
    private String atividade;

    public Funcionario(String nome, String atividade) {
        this.nome = nome;
        this.atividade = atividade;
    }

    public String getNome(){
        return nome;
    }

    public String getAtividade(){
        return atividade;
    }

    public String toString() {
        return "\nNome: " + getNome() + "\nAtividade: " + getAtividade();
    }
}

File: PHP_atividade1/classes/Funcionario.php
<?php

class Funcionario
{
    private String $nome;
    private String $atividade;

    public function __construct(string $nome, string $atividade)
    {
        $this->nome = $nome;
        $this->atividade = $atividade;
    }

    public function getNome()
    {
        return $this->nome;
    }

    public function getAtividade()
    {
        return $this->atividade;
    }
}